### PR TITLE
[Core] eigen_solver_tests adding skiptest for ComplexPardisoLUSolver

### PR DESCRIPTION
**Pull request by philbucher**
otherwise the test fails

### CODE REVIEW

**Comment by armingeiser** [comment]
Thanks, i forgot this

**Review by armingeiser — APPROVED** [review]

File: kratos/tests/test_eigen_solvers.py [hunk 1]
```diff
@@ -188,6 +188,8 @@ def test_FEAST_with_mkl_solver(self):
             self.skipTest("KratosMultiphysics.ExternalSolversApplication is not available")
         try:
             import KratosMultiphysics.EigenSolversApplication
+            if not hasattr(KratosMultiphysics.EigenSolversApplication, "ComplexPardisoLUSolver"):
```
Comment (RiccardoRossi) [inline kratos/tests/test_eigen_solvers.py:191]: i think you should add this prior to line 190

**Comment by RiccardoRossi** [comment]
just to add that in FullDebug the eigensolvers tests are taking too long

File: kratos/tests/test_eigen_solvers.py (hunk 1 above)
Comment (armingeiser) [reply kratos/tests/test_eigen_solvers.py:191]: No this is ok, the EigenSolversApplication availability is checked in the lines above.
This is just for the case that EigenSolversApplication is compiled without MKL.

Comment (RiccardoRossi) [reply kratos/tests/test_eigen_solvers.py:191]: ok sorry

**Pull request merged**